backend/.env

```diff
@@ -1,5 +1,5 @@
-#DATABASE = "mongodb://localhost:27017"
-#RESEND_API = "your resend_api"
+DATABASE = "mongodb+srv://lohahussain0:0CaBBqrHJ3ZeIHfc@officeapi.yq9k97d.mongodb.net/idurtest?retryWrites=true&w=majority"
```

Comment:
Don't commit your private connection string with credentials

Reply:
Got it. I removed it.